src/cgame/default/cg_main.c

```diff
@@ -106,6 +107,8 @@ static void Cg_Init(void) {
 
 	cg_fov = cgi.Cvar("cg_fov", "110.0", CVAR_ARCHIVE, NULL);
 	cg_fov_zoom = cgi.Cvar("cg_fov_zoom", "55.0", CVAR_ARCHIVE, NULL);
+	cg_fov_interpolate = cgi.Cvar("cg_fov_interpolate", "100", CVAR_ARCHIVE,
+			"Speed to change the fov at in x100 degrees/second. 0 disables, 1 is 100 deg/sec, 0.5 is 50 deg/sec");
```

Comment:
Yeah, the alias has always seemed slightly weird to me, ideally it would just be a normal command (ideally one that just gets eaten locally by cgame without making it to the server). But that's scope creep on what you are trying to achieve :) It just has the advantage of not being dependent on the binds restoring fov.